pywifi/wifiutil_abc.py

```diff
@@ -7,14 +7,17 @@
 from abc import ABCMeta, abstractmethod
 
 
-class WifiUtilABC(metaclass=ABCMeta):
+class WifiUtilABC():
     """
     Abstract class used for each OS implementation.
     
     To support the wifi functions in each OS, implementors should
     finish all the abstract method defined here.
     """
-
+    
+    
+    __metaclass__=ABCMeta #Use old style of defining metaclasses to support Py2
```

Comment:
I found this will cause the abstract class property disappear in python 3.5. So, I followed the solution in http://stackoverflow.com/questions/35673474/using-abc-abcmeta-in-a-way-it-is-compatible-both-with-python-2-7-and-python-3-5 finally.